Model/Payment.php

```diff
@@ -385,7 +386,7 @@ public function capture(InfoInterface $payment, $amount)
                 );
             }
 
-            $captureAmount = $amount * 100;
+            $captureAmount = CurrencyUtils::toMinor($amount, "USD");
```

Comment:
Shouldn't these be `toMinorWithoutRounding`?

Reply:
Bolt API only accepts integer so this should be `toMinor` (but you're right previous logic didn't have rounding)